packages/zowe-explorer/src/job/actions.ts

```diff
@@ -248,63 +249,110 @@ export async function setPrefix(job: IZoweJobTreeNode, jobsProvider: IZoweTree<I
  *
  * @param jobsProvider The tree to which the node belongs
  */
-export async function deleteCommand(job: IZoweJobTreeNode, jobsProvider: IZoweTree<IZoweJobTreeNode>) {
-    const nodesToDelete: string[] = [];
-    const deletedNodes: string[] = [];
-    const selectedNodes: IZoweJobTreeNode[] = jobsProvider.getTreeView().selection;
-    const nodes: IZoweJobTreeNode[] = selectedNodes.filter(
-        (jobNode) => jobNode.job !== undefined && jobNode.job !== null
-    );
+export async function deleteCommand(
+    jobsProvider: IZoweTree<IZoweJobTreeNode>,
+    job?: IZoweJobTreeNode,
+    jobs?: IZoweJobTreeNode[]
+) {
+    if (jobs && jobs.length) {
+        await deleteMultipleJobs(
+            jobs.filter((jobNode) => jobNode.job !== undefined && jobNode.job !== null),
+            jobsProvider
+        );
+        return;
+    }
+    if (job) {
+        await deleteSingleJob(job, jobsProvider);
+        return;
+    }
+}
 
-    if (nodes.length > 0) {
-        for (const node of nodes) {
-            nodesToDelete.push(`${node.job.jobname}(${node.job.jobid})`);
-        }
-    } else if (job) {
-        nodesToDelete.push(`${job.job.jobname}(${job.job.jobid})`);
+const deleteSingleJob = async (job: IZoweJobTreeNode, jobsProvider: IZoweTree<IZoweJobTreeNode>): Promise<void> => {
```

Comment:
Can you please change this into our standard format? something like this `export async function deleteSingleJob`. At least i want our functions to be coded in the same way. this is supposed to be a function, right?

Reply:
Yeah, sure, I can do that, thank you for bringing this up!